docs/marionette.view.md

```diff
@@ -333,9 +333,9 @@ bound at the time of instantiation, and an exception will be thrown
 if the handlers on the view do not exist.
 
 The `modelEvents` and `collectionEvents` will be bound and
-unbound with the Backbone.View `delegateEvents` and `undelegateEvents`
-method calls. This allows the view to be re-used and have
-the model and collection events re-bound.
+unbound with the Backbone.View `delegateEntityEvents` and `undelegateEntityEvents`
+method calls. `delegateEntityEvents` is called in the View's `constructor` and
+entity events are unbound during the View's `destroy`.
```

Comment:
yeah some people totally wont like this, for long lived views... wonder if this should respect preventDestroy ?